### ISSUE TITLE
Pyright v1.1.340 crashes with an internal error

### ISSUE THREAD

**Issue by agronholm** [microsoft/pyright#6722]
I'm running `pyright --verifytypes` against https://github.com/agronholm/anyio and it started failing with an internal error as soon as v1.1.340 was published. After downgrading to v1.1.339 it works fine.
<details>
<summary>Run pyright against AnyIO</summary>

Run pyright --verifytypes anyio

```
added 1 package, and audited 2 packages in 2s

found 0 vulnerabilities
Module name: "anyio"
Package directory: "file:///home/runner/work/anyio/anyio/src/anyio"
Module directory: "file:///home/runner/work/anyio/anyio/src/anyio"
Path of py.typed file: "file:///home/runner/work/anyio/anyio/src/anyio/py.typed"

Public modules: 1
   anyio
   error: An internal error occurred while verifying types: "TypeError: Cannot read properties of undefined (reading 'startsWith')
      at Object.file (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/common/uri/uri.ts:167:33)
      at forEach (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/packageTypeVerifier.ts:592:38)
      at Map.forEach (<anonymous>)
      at b._getTypeKnownStatusForSymbolTable (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/packageTypeVerifier.ts:515:21)
      at b._verifyTypesOfModule (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/packageTypeVerifier.ts:398:22)
      at forEach (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/packageTypeVerifier.ts:185:30)
      at Array.forEach (<anonymous>)
      at b.verify (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/packageTypeVerifier.ts:184:35)
      at verifyPackageTypes (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/pyright.ts:493:33)
      at processArgs (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/pyright.ts:370:16)
      at main (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/pyright.ts:970:28)
      at Object.call (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright/src/pyright.ts:3:1)
      at __webpack_require__ (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/bootstrap:19:32)
      at fn (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/bootstrap:35:79)
      at Function.O (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/runtime/chunk loaded:23:12)
      at next (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/bootstrap:36:44)
      at Function.x (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/runtime/startup chunk dependencies:5:9)
      at /home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/webpack/startup:2:47
      at Object.<anonymous> (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright.js:1:2038)
      at Module._compile (node:internal/modules/cjs/loader:1356:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
      at Module.load (node:internal/modules/cjs/loader:[11](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:12)97:32)
      at Function.Module._load (node:internal/modules/cjs/loader:1013:[12](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:13))
      at Module.require (node:internal/modules/cjs/loader:1225:19)
      at require (node:internal/modules/helpers:177:18)
      at Object.<anonymous> (/home/runner/.cache/pyright-python/1.1.340/node_modules/pyright/index.js:8:1)
      at Module._compile (node:internal/modules/cjs/loader:[13](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:14)56:[14](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:15))
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
      at Module.load (node:internal/modules/cjs/loader:1[19](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:20)7:32)
      at Function.Module._load (node:internal/modules/cjs/loader:1013:12)
      at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:1[28](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:29):12)
      at node:internal/main/run_main_module:28:[49](https://github.com/agronholm/anyio/actions/runs/7196027422/job/19599934378?pr=651#step:6:50)"

Symbols used in public interface:

Symbols exported by "anyio": 0
  With known type: 0
  With ambiguous type: 0
  With unknown type: 0
  Functions without docstring: 0
  Functions without default param: 0
  Classes without docstring: 0

Other symbols referenced but not exported by "anyio": 0
  With known type: 0
  With ambiguous type: 0
  With unknown type: 0

Type completeness score: 0%

Completed in 1.499sec
```

</details>

**Comment by erictraut** [comment]
@rchiodo, it looks like you introduced a fix for this, right? Should we tag it as "addressed in next version"?

**Comment by rchiodo** [comment]
Yeah sorry, missed adding the label.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.341, which I just published.
